src/benchmarklib/abstract_table_generator.cpp

```diff
@@ -354,7 +327,77 @@ std::shared_ptr<BenchmarkConfig> AbstractTableGenerator::create_benchmark_config
   return std::make_shared<BenchmarkConfig>(config);
 }
 
+void AbstractTableGenerator::_create_chunk_indexes(
+    std::unordered_map<std::string, BenchmarkTableInfo>& table_info_by_name) {
+  Timer timer;
+  std::cout << "- Creating chunk indexes" << std::endl;
+  const auto& indexes_by_table = _indexes_by_table();
+  if (indexes_by_table.empty()) {
+    std::cout << "-  No indexes defined by benchmark" << std::endl;
```

Comment:
If we don't return here, we'll output the last line which states how long it took to create indexes.